File: test/single-language/rust/cargo/http-service/failure-configuration-rejected/src/service.rs
use crate::model::Summary;
pub fn summarize(raw: Option<&str>) -> Result<Summary, &'static str> {
    let mut items = Vec::new();
    for token in raw.unwrap_or("1,2,3").split(',') {
        if token.is_empty() || token.len() > 10 || !token.bytes().all(|c| c.is_ascii_digit()) || items.len() == 20 {
            return Err("invalid-values");
        }
        let item = token.parse::<u32>().map_err(|_| "invalid-values")?;
        if item > 10000 { return Err("invalid-values"); }
        items.push(item);
    }
    Ok(Summary::new(items))
}

File: test/single-language/python/stdlib/http-service/success-runtime-config/http_service_fixture/service.py
import re
from .model import Summary


def summarize(raw):
    if raw is None:
        items = [1, 2, 3]
    else:
        tokens = raw.split(',')
        if len(tokens) > 20 or any(not re.fullmatch(r'[0-9]{1,10}', token) for token in tokens):
            raise ValueError('invalid-values')
        items = [int(token) for token in tokens]
    if not 1 <= len(items) <= 20 or any(not 0 <= item <= 10000 for item in items):
        raise ValueError('invalid-values')
    return Summary(tuple(items))
